crowdloan/kusama.json

```diff
@@ -204,8 +204,8 @@
     "name": "Picasso",
     "token": "PICA",
     "description": "Cross-chain interoperable infrastructure",
-    "website": "https://www.composable.finance/",
-    "icon": "https://raw.githubusercontent.com/polkadot-js/apps/master/packages/apps-config/src/ui/logos/nodes/picasso.png",
+    "website": "https://picasso.composable.finance/",
+    "icon": "https://raw.githubusercontent.com/polkadot-js/apps/master/packages/apps-config/src/ui/logos/nodes/picasso.svg",
```

Comment:
that one gives me 404 as well, and also I think you have an icon for Picasso in this PR adding it to our repo, so why would we need to have an icon from Polkadot JS one?